jax/numpy/__init__.pyi

```diff
@@ -13,6 +13,8 @@ from jax._src.typing import (
     Array, ArrayLike, DType, DTypeLike,
     DimSize, DuckTypedArray, Shape, DeprecatedArg
 )
+from jax._src.sharding import Sharding
+from jax._src.lib import xla_client as xc
```

Comment:
Unfortunately, this import here causes failures in pytype because it doesn't know how to resolve `xla_client`, so we use `_Device` as defined below.